app/javascript/tests/close_contact/CloseContact.test.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme';
+import { Button, Modal } from 'react-bootstrap';
+import InfoTooltip from '../../components/util/InfoTooltip';
+import CloseContact from '../../components/close_contact/CloseContact.js'
+import { mockPatient1 } from '../mocks/mockPatients'
+import { mockCloseContact1, mockCloseContact2 } from '../mocks/mockCloseContact'
+
+const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
+const ASSIGNED_USERS = [ 123234, 512678, 910132 ]
+const testInputValues = [
+  { field: 'first_name', value: 'Anthony', inputType: 'FormControl' },
+  { field: 'last_name', value: 'Stark', inputType: 'FormControl' },
+  { field: 'primary_telephone', value: '1234567890', inputType: 'PhoneInput' },
+  { field: 'email', value: 'tinman@starkindustries.com', inputType: 'FormControl' },
+  { field: 'last_date_of_exposure', value: '2020-05-16', inputType: 'DateInput' },
+  { field: 'assigned_user', value: ASSIGNED_USERS[0], inputType: 'FormControl' },
+  { field: 'notes', value: 'Inevitable', inputType: 'FormControl' },
+]
+
+function getShallowWrapper(closeContact) {
+  return shallow(<CloseContact authenticity_token={authyToken} patient={mockPatient1} close_contact={closeContact}
+    can_enroll_patient_close_contacts={true} assigned_users={ASSIGNED_USERS}  />);
+}
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('CloseContact', () => {
+  it('Properly renders all main components for empty close contact', () => {
```

Comment:
You should also check in this case that the proper actions are rendered. Edit, Contact Attempt, and Enroll, and NOT View Record.

Reply:
Those would not be rendered if there were no close contacts. That's not how the component is set up.

Reply:
I think I originally misunderstood the purpose of this test. There should be a test that verifies that Enroll shows up instead of View Record when they are not already enrolled.

Reply:
Good catch! Adding an additional mock and a test.